src/main/java/org/opentripplanner/inspector/raster/AreaEdgeRenderer.java

```diff
@@ -0,0 +1,105 @@
+package org.opentripplanner.inspector.raster;
+
+import java.awt.Color;
+import java.util.Iterator;
+import java.util.Optional;
+import org.opentripplanner.inspector.raster.EdgeVertexTileRenderer.EdgeVertexRenderer;
+import org.opentripplanner.inspector.raster.EdgeVertexTileRenderer.EdgeVisualAttributes;
+import org.opentripplanner.inspector.raster.EdgeVertexTileRenderer.VertexVisualAttributes;
+import org.opentripplanner.street.model.edge.AreaEdge;
+import org.opentripplanner.street.model.edge.AreaEdgeList;
+import org.opentripplanner.street.model.edge.Edge;
+import org.opentripplanner.street.model.vertex.IntersectionVertex;
+import org.opentripplanner.street.model.vertex.OsmBoardingLocationVertex;
+import org.opentripplanner.street.model.vertex.TransitBoardingAreaVertex;
+import org.opentripplanner.street.model.vertex.TransitEntranceVertex;
+import org.opentripplanner.street.model.vertex.TransitPathwayNodeVertex;
+import org.opentripplanner.street.model.vertex.TransitStopVertex;
+import org.opentripplanner.street.model.vertex.Vertex;
+
+/**
+ * Render OSM areas only. Show area edge vertices and OSM boarding locations. Highlight visiblity point vertices.
+ *
+ */
+public class AreaEdgeRenderer implements EdgeVertexRenderer {
```

Comment:
Should we have this as a `GraphInspectorVectorTileResource#DEBUG_LAYERS` instead of a raster layer?

Reply:
I think we should totally do that. I didn't expect Vesa to do that since it's quite a bit of work.

But yes, I think the goal should be make creating the vector layers as easy as the raster ones.